environments/icds-cas/public.yml

```diff
@@ -3,8 +3,8 @@ daily_deploy_email: tech-announce-daily@dimagi.com
 
 elasticsearch_endpoint: '{{ groups.es0.0 }}'
 elasticsearch_cluster_name: 'icds-2.0'
-elasticsearch_version: 1.7.6
-elasticsearch_download_sha256: 78affc30353730ec245dad1f17de242a4ad12cf808eaa87dd878e1ca10ed77df.
+elasticsearch_version: 2.4.6
+elasticsearch_download_sha256: 5f7e4bb792917bb7ffc2a5f612dfec87416d54563f795d6a70637befef4cfc6f.
```

Comment:
What's with these changes?  Did you intend for them to be in a different PR?

Reply:
I will remove from this PR